projects/hydro-protocol/index.js

```diff
@@ -1,10 +1,65 @@
-const { queryContract } = require("../helper/chain/cosmos");
+const { queryContract, totalSupply } = require("../helper/chain/cosmos");
 const ADDRESSES = require("../helper/coreAssets.json");
+const { getClient } = require("../helper/chain/injective");
+const { IndexerGrpcMitoApi, spotPriceFromChainPrice, spotQuantityFromChainQuantity, IndexerGrpcSpotApi } = require('@injectivelabs/sdk-ts')
+const BigNumber = require("bignumber.js");
+const { get } = require("../helper/http");
+
+const hinj = "inj18luqttqyckgpddndh8hvaq25d5nfwjc78m56lc"
+const autoCompound = "inj1mjcg8a73904rj4w7t5qkgn0apua98n059nufma"
+const hdro = "factory/inj1etz0laas6h7vemg3qtd67jpr6lh8v7xz7gfzqw/hdro"
+const xhdro = "inj1qc2tw477wwuvkad0h3g78xqgwx4k8knat6vz0h"
+const hdroInjMarket = "0xc8fafa1fcab27e16da20e98b4dc9dda45320418c27db80663b21edac72f3b597"
+
+const injectiveSpotApi = new IndexerGrpcSpotApi("https://sentry.exchange.grpc-web.injective.network:443")
+const injectiveClient = getClient()
 
-const autocompoundContract = 'inj1mjcg8a73904rj4w7t5qkgn0apua98n059nufma'
 async function tvl(_, _1, _2, { api }) {
-  const { total_supply } = await queryContract({ chain: api.chain, contract: "inj18luqttqyckgpddndh8hvaq25d5nfwjc78m56lc", data: { token_info: {} } })
-  api.add(ADDRESSES.injective.INJ, total_supply)
+  const { total_supply } = await queryContract({ chain: api.chain, contract: hinj, data: { token_info: {} } })
+  const { total_bonded } = await queryContract({ chain: api.chain, contract: autoCompound, data: { state: {} } })
+  const hdroCirculatingSupply = new BigNumber(await get("https://index.hydroprotocol.finance/circulating-supply"))
+  const marketInjQuantity = await loadMarketInjQuantity()
+  const hdroInjPrice = await loadHdroInjPrice()
+
+  api.add(
+    ADDRESSES.injective.INJ,
+    new BigNumber(total_supply)
+      .plus(new BigNumber(total_bonded))
+      .plus(new BigNumber(marketInjQuantity))
```

Comment:
why is this counted as your product tvl?

Reply:
I fixed it to marketINJQuantity -> marketHdroQuantity

Reply:
Locked hrdo can be counted under staking, unsure why you are trying to count hrdo/inj in order book as tvl

Reply:
Because it is our liquidity

Reply:
Yes, but that is not what we consider as tvl, tvl is value locked into your protocol/contracts but the liquidity value belongs to the Dex/orderbook protocol

Reply:
Ok thanks. I'm going to remove it